login-fire-form-behavior.html

```diff
@@ -103,6 +104,21 @@
     },
 
     /**
+     * Store Crendtials in Credentials API
+     */
+    storeCredentials: function(email, password) {
+      if (navigator.credentials) {
+        var cred = new PasswordCredential({
```

Comment:
I don't understand why a new credential is created here when the Firebase Auth API provides a fresh one at [this line](https://github.com/convoo/login-fire/pull/135/commits/a6a0c578291c206a591c6f56c7f98399bae68a64#diff-e2bd87c7eceee24aa6d08641240914dbR397). It is accessible on the `signedin` event.

Reply:
PasswordCredential is a class from the browser needed to store it in the Credential API (https://developer.mozilla.org/fr/docs/Web/API/PasswordCredential)
Didn't see an object from this class in the response of the Firebase Auth API, so I had to create it myself with the email/password.

Reply:
Got it.